library/std/src/os/solid/mod.rs

```diff
@@ -0,0 +1,3 @@
+//! SOLID-specific definitions
+
+#![stable(feature = "raw_ext", since = "1.1.0")]
```

Comment:
1. This likely shouldn't be stable.
2. As it is empty, it can probably be omitted.

Reply:
This module was removed in 0595f091f6454ad47e7183ed3af6f8cbb5b9ee95.